homeassistant/components/switchbee/entity.py

```diff
@@ -83,3 +87,21 @@ async def async_refresh_state(self) -> None:
             return
         except SwitchBeeError:
             return
+
+    def _check_if_became_offline(self) -> None:
+        """Check if the device was online (now offline), log message and mark it as Unavailable."""
+        if self._is_online:
```

Comment:
Should `async_refresh_state` not be called in the base class?
```suggestion
        # This specific call will refresh the state of the device in the CU
        self.hass.async_create_task(self.async_refresh_state())

        if self._is_online:
```